src/app/handlers/button_handler.ts

```diff
@@ -47,9 +47,18 @@ export async function call(interaction: ButtonInteraction<CacheType>) {
         const param_q = params.get('q') || null;
         const param_d = params.get('d') || null;
         const param_t = params.get('t') || null;
-        const voiceLockerIds = ['voiceLock_inc', 'voiceLock_dec', 'voiceLockOrUnlock'];
-        if (voiceLockerIds.includes(interaction.customId)) {
-            await voiceLockerUpdate(interaction);
+        const voiceLockeCommandIds = ['voiceLock_inc', 'voiceLock_dec', 'voiceLockOrUnlock'];
+        const voiceLockIds = [
+            'voiceLock_dec1',
+            'voiceLock_dec10',
+            'voiceLock_inc1',
+            'voiceLock_inc10',
+            'LockSwitch',
```

Comment:
lowerCamelに統一とおもったけど、 `voiceLock_inc10`とかもうぐちゃぐちゃでどうでもよくなったw
んー、あとでまとめてenum化したいかも。 `constant.ts` あたりに
```suggestion
            'lockSwitch',
```

Reply:
stgにマージしたあと別ブランチで全部のボタンのID統一したいので一旦保留で。